applications/exposure-checker/values-usdfdev.yaml

```diff
@@ -0,0 +1,19 @@
+image:
+  tag: tickets-DM-48431
+
+environment:
+  PGPASSFILE: "/opt/lsst/butler/secrets/postgres-credentials.txt"
+  PGUSER: "rubin"
+  AWS_SHARED_CREDENTIALS_FILE: "/opt/lsst/butler/secrets/aws-credentials.ini"
+  S3_ENDPOINT_URL: "https://s3dfrgw.slac.stanford.edu"
+  LSST_RESOURCES_S3_PROFILE_embargo: "https://sdfembs3.sdf.slac.stanford.edu"
+  LSST_DISABLE_BUCKET_VALIDATION: "1"
+
+config:
+  butler_repo: "s3://embargo@rubin-summit-users/butler.yaml"
+  butler_collection: "u/kadrlica/binCalexp4"
+  s3_profile_name: "rubin-rubintv-data-summit"
+  s3_endpoint_url: "https://s3dfrgw.slac.stanford.edu"
```

Comment:
This duplicates a value in the `environment` mapping; can they be consolidated?

Reply:
Thanks, there were multiple cases of copy-paste sloppiness that I should have caught beforehand.